en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+      <source xml:space="preserve">Mozilla VPN</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/settings/ViewAboutUs.qml</context><context context-type="linenumber">52</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewInitialize.qml</context><context context-type="linenumber">30</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/settings/ViewAboutUs.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="tos">
+      <source xml:space="preserve">Terms of Service</source>
+      <context-group purpose="location"><context context-type="linenumber">21</context></context-group>
+    </trans-unit>
+    <trans-unit id="privacyPolicy">
+      <source xml:space="preserve">Privacy Policy</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+    </trans-unit>
+    <trans-unit id="aboutUs">
+      <source xml:space="preserve">About us</source>
+      <context-group purpose="location"><context context-type="linenumber">34</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/settings/ViewSettingsMenu.qml</context><context context-type="linenumber">39</context></context-group>
+    </trans-unit>
+    <trans-unit id="productDescription">
+      <source xml:space="preserve">A fast, secure and easy to use VPN. Built by the makers of Firefox.</source>
+      <context-group purpose="location"><context context-type="linenumber">57</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewInitialize.qml</context><context context-type="linenumber">32</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/main.cpp</context><context context-type="linenumber">42</context></context-group>
+    </trans-unit>
+    <trans-unit id="releaseVersion">
+      <source xml:space="preserve">Release Version</source>
+      <context-group purpose="location"><context context-type="linenumber">65</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/settings/ViewLanguage.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="language">
+      <source xml:space="preserve">Language</source>
+      <context-group purpose="location"><context context-type="linenumber">18</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/settings/ViewSettingsMenu.qml</context><context context-type="linenumber">33</context></context-group>
+    </trans-unit>
+    <trans-unit id="system">
+      <source xml:space="preserve" />
```

Comment:
What's up with the empty string here? There's another one right after, and a couple more at the end of the file.